README.md

```diff
@@ -1,2 +1,182 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Create Enumerable <a name="about-project"></a>
+
+**Create Enumerable** is a program to check certain conditions on a list. 
+
+- Check all. 
+- Check any.
+- Filter
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>N/A</li>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Check all
+- Check any
+- Filter
+
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LIVE DEMO -->
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Getting Started -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To run the project locally, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+
+- Install [Git](https://git-scm.com/)
+- Install [Node](https://nodejs.org/en/)
+- Install a code editor, I suggest [VsCode](https://code.visualstudio.com/)
+
+### Setup
+
+Clone the repo into your local machine:
+
+```bash
+git clone git@github.com:edgardsoza/enumerable/tree/enumerable.git
+```
+
+Install the dependencies using npm:
+```bash
+npm install
+```
+
+### Usage
+
+To start the development server run
+```bash
+npm run start
+```
+- this command will start the development server on `localhost:8080` address
+
+To create and optimized production build run
+```bash
+npm run build
+```
+- this command will create a bunduled chunks of code inside `dist/` folder
+
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Santiago Pons Lezica**
+
+- GitHub: [@sntgopl](https://github.com/sntgopl)
+- LinkedIn: [Santiago Pons Lezica](https://www.linkedin.com/in/spons-lezica7/?originalSubdomain=ar)
+
+👤 **Edgard Soza**
+
+- [GitHub](https://github.com/edgardsoza)
+- [LinkedIn](https://www.linkedin.com/in/edgard-soza-sobalvarro/)
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Create more filters or options.
+- Add frontend code
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Contributing -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
```

Comment:
- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```